lib/routes/charge/pos_invoice.dart

```diff
@@ -233,31 +233,59 @@ class POSInvoiceState extends State<POSInvoice> {
                                                 child: Align(
```

Comment:
This widget needs to be aligned with the item icons.  Add left padding of ~(8-16)dp depending on the inner padding on the icon.

Reply:
Good one.